katalog/istio-operator/jaeger-operator/BREAKING.md

```diff
@@ -0,0 +1,44 @@
+## Comprehensive Upgrade Guide for the Jaeger Operator from v1.22 to the Latest Release 🦮
+
+### Introduction
+This guide provides detailed instructions for upgrading the Jaeger Operator component in a Service Mesh from version 1.22 to the latest release. The previous version is substantially outdated, and a direct upgrade path is unavailable. As a result, it is necessary to completely remove the old version before installing the new one. This document explains the reasons for this approach and outlines the steps to successfully transition to the latest version, which can be explored in detail [here](https://www.jaegertracing.io/docs/1.57/operator/).
+
+### Breaking Changes Notification 💔
+
+#### Key Points
+- **Version Obsolescence:** The Jaeger Operator version 1.22 is no longer supported, and direct updates to the new version are not possible.
+- **Impact on Cluster Operations:** The removal of the old version and installation of the new one may temporarily affect the stability and functionality of your Service Mesh.
```

Comment:
out of curiosity, why may removing Jeager operator affect the service mesh? Isn't it used just for visualization?

Reply:
@stefanoghinelli could provide a better answer, but according to the procedure, the previous version needs to be removed in order to install the new one, which could impact the operations in the cluster. The Jeager Operator and Operated would be directly affected. This is a breaking change because some custom implementations related to Jaeger could be affected, and we don't have any direct upgrade path for this. As for the impact on the service mesh, I'm not entirely sure; perhaps @stefanoghinelli  can provide more details about it.